e2e/content-services/permissions-component.e2e.ts

```diff
@@ -25,93 +25,269 @@ import AlfrescoApi = require('alfresco-js-api-node');
+
+        await alfrescoJsApi.core.nodesApi.updateNode(roleCoordinatorFolder.entry.id,
+            {
+                permissions: {
+                    locallySet: [{
+                        authorityId: coordinatorUser.getId(),
+                        name: 'Coordinator',
+                        accessStatus: 'ALLOWED'
+                    }]
+                }
+            });
+
+        await alfrescoJsApi.core.nodesApi.updateNode(roleContributorFolder.entry.id,
+            {
+                permissions: {
+                    locallySet: [{
+                        authorityId: contributorUser.getId(),
+                        name: 'Contributor',
+                        accessStatus: 'ALLOWED'
+                    }]
+                }
+            });
+
+        await alfrescoJsApi.core.nodesApi.updateNode(roleEditorFolder.entry.id,
+            {
+                permissions: {
+                    locallySet: [{
+                        authorityId: editorUser.getId(),
+                        name: 'Editor',
+                        accessStatus: 'ALLOWED'
+                    }]
+                }
+            });
+
+        await alfrescoJsApi.core.nodesApi.updateNode(privateSiteFile.entry.id,
+            {
+                permissions: {
+                    locallySet: [{
+                        authorityId: managerUser.getId(),
+                        name: 'SiteConsumer',
+                        accessStatus: 'ALLOWED'
+                    }]
+                }
+            });
+
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'RoleConsumer' + fileModel.name, roleConsumerFolder.entry.id);
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'RoleContributor' + fileModel.name, roleContributorFolder.entry.id);
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'RoleCoordinator' + fileModel.name, roleCoordinatorFolder.entry.id);
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'RoleCollaborator' + fileModel.name, roleCollaboratorFolder.entry.id);
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'RoleEditor' + fileModel.name, roleEditorFolder.entry.id);
+        await uploadActions.uploadFile(alfrescoJsApi, fileModel.location, 'Site' + fileModel.name, siteFolder.entry.id);
+        browser.driver.sleep(15000); // wait search get the groups, files and folders
 
         done();
     });
 
     afterAll(async (done) => {
         await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
         await alfrescoJsApi.core.sitesApi.deleteSite(publicSite.entry.id);
-        await alfrescoJsApi.core.groupsApi.deleteGroup(group.entry.id);
+        await alfrescoJsApi.core.sitesApi.deleteSite(privateSite.entry.id);
+        await alfrescoJsApi.core.groupsApi.deleteGroup(groupId);
+        await uploadActions.deleteFilesOrFolder(alfrescoJsApi, roleConsumerFolder.entry.id);
```

Comment:
Same comment as above. All the folders, files could be part of an array or a map, in this way you can use loops in order to delete them, etc.